docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+
+services:
+  postgres:
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_DB: openousd
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: openousd
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
this makes it so you don't need a password to connect. Generally not good, but the whole point is that this will run locally so that we don't need to pay for hosting.